### PR TITLE
173108170 activity home page

### PR DESCRIPTION
**Pull request by mklewandowski**
Add LARA activity intro/home page based on activity JSON.  This PR includes:
- add SVG icons used in intro page
- add mock activity data used to simulate displaying activities 
- add structure and logo images for project types (logos are used in intro header)
- add React components used to display intro page content (headers, main content, page nav, etc.)
- add basic page navigation with stub content shown for activity pages

### CODE REVIEW

File: src/components/activity-header/header.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import './header.scss';
+
+interface IProps {
+  projectId: number | null;
+}
+
+export class Header extends React.PureComponent <IProps> {
+  render() {
+    const projectType = ProjectTypes.find(pt => pt.id === this.props.projectId);
+    const logo = projectType !== undefined ? projectType.logo : undefined;
```
Comment (kswenson) [inline src/components/activity-header/header.tsx:14]:
```suggestion
    const logo = projectType?.logo;
```

File: src/components/activity-header/header.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    const projectURL = projectType !== undefined ? projectType.url : "";
```
Comment (kswenson) [inline src/components/activity-header/header.tsx:15]:
```suggestion
    const projectURL = projectType?.url || "";
```

File: src/components/activity-header/header.tsx (hunk 1 above)
Comment (mklewandowski) [reply src/components/activity-header/header.tsx]: Fixed.

File: src/components/activity-header/header.tsx (hunk 2 above)
Comment (mklewandowski) [reply src/components/activity-header/header.tsx]: Fixed.

**Comment by mklewandowski** [comment]
@kswenson I'm not opposed to either of those general ideas.  
- A few of the components should probably be converted from class to functional components just to simplify them.  
- I modeled my component unit tests after an example in the starter projects.  If React testing library is what we want to go with, maybe we should consider removing the enzyme-based unit test from the starter projects and using React testing library (testing-library/react-hooks is included in the starter projects, but testing-library/react is not)?  

However, in hopes of getting this PR in, I would also advocate doing the above conversions in smaller ongoing PRs when time permits.

**Pull request merged**